bin/11_1/src/Main.cpp

```diff
@@ -0,0 +1,25 @@
+#include <cctype>
+#include <exception>
+#include <iostream>
+#include <fstream>
+
+int main() {
+  try {
+    std::ifstream inputFile { "files/Input.txt" };
+    std::ofstream outputFile { "files/Out.txt" };
+    if (inputFile.is_open() && outputFile.is_open()) {
```

Comment:
Prefer the next syntax to avoid deep indentations in your code:
if (!inputFile.is_open() || !outputFile.is_open())
    throw std::exception .....

Please read the next article
https://www.fluentcpp.com/2018/02/20/on-using-guards-in-cpp/